### DISCUSSION TITLE
will the argo-ui support less-code workflow write?

### DISCUSSION THREAD

**Discussion by leveryd** [argoproj/argo-workflows#10780] [General]
currently we must write yaml as workflow, will u plan to support we write workflow just by drag and drop on argo-ui?

**Comment by tooptoop4** [comment]
these are some very early ways: https://equinor.github.io/flowify-documentation/ or https://github.com/corpulent/visual-argo-workflows

**Reply by leveryd** [reply]
I see, but those ways are not enough powerful.
